src/client/components/Molecules/HeroSearchForm/HeroSearchForm.jsx

```diff
@@ -0,0 +1,26 @@
+import { Link } from 'react-router-dom';
+
+import css from './HeroSearchForm.module.css';
+
+
+const HeroSearchForm = ({ value, onChange }) => {
+    return (
+      <form className={css.formSearch}>
+        <div className={css.formSearchLabel}>
+          <input
+            type="text"
+            value={value}
+            onChange={e => onChange(e.target.value)}
+            placeholder="Beef"
```

Comment:
I suggest you to change this `placeholder` on sometihng like "category" because we can mislead our pottential client.